hyrax.gemspec

```diff
@@ -56,6 +56,7 @@ SUMMARY
   spec.add_dependency 'iiif_manifest', '>= 0.3', '< 0.6'
   spec.add_dependency 'jquery-datatables-rails', '~> 3.4'
   spec.add_dependency 'jquery-ui-rails', '~> 6.0'
+  spec.add_dependency 'jsonpath', '~> 0.5.8'
```

Comment:
I'm hoping that, in the long run, we can do this without adding the `jsonpath` dependency.